deployments/deploy-ssh.sh

```diff
@@ -239,6 +239,13 @@ if [ -n "${TORPEDO_SSH_KEY}" ]; then
     TORPEDO_SSH_KEY_MOUNT="{ \"name\": \"ssh-key-volume\", \"mountPath\": \"/home/torpedo/\" }"
 fi
 
+ORACLE_API_KEY_VOLUME=""
+if [ -n "${ORACLE_API_KEY}" ]; then
+    kubectl create secret generic key4oracle --from-file=${ORACLE_API_KEY}
+    ORACLE_API_KEY_VOLUME="{ \"name\": \"oracle-api-key-volume\", \"secret\": { \"secretName\": \"key4oracle\", \"defaultMode\": 256 }}"
```

Comment:
can we move this to spawn or in torpedo initialization rather that having in deploy-ssh

Reply:
Raised https://portworx.atlassian.net/browse/PTX-13904 to create this secret via spawn.